### DISCUSSION TITLE
How to show markers at all pointConnector in linechart ?

### DISCUSSION THREAD

**Discussion by dev-mihirahir** [patrykandpatrick/vico#490] [Questions] [answered]
I need assistance to achieve the functionality shown in the screenshot below.
![Screenshot 2023-12-13 at 12 09 29 PM](https://github.com/patrykandpatrick/vico/assets/108797292/f9051d92-b37d-4893-a042-af8e44fd687b)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Such points can be added via `LineSpec`. The `LineSpec` constructor and the `lineSpec` composable function both have `point` parameters and parameters for the point size.

**Reply by dev-mihirahir** [reply]
Thanks, @patrickmichalik 
It's work for me.

**Reply by patrickmichalik** [reply]
You’re welcome!
